packages/editor/src/client/api/index.ts

```diff
@@ -544,6 +544,7 @@ export type MutationUpdateUserArgs = {
 export type MutationResetUserPasswordArgs = {
   id: Scalars['ID'];
   password: Scalars['String'];
+  sendMail?: Maybe<Scalars['Boolean']>;
```

Comment:
Why optional boolean?

Reply:
Good point. I made the boolean optional because i wanted the `resetUserPassword` mutation to be backwards compatible. But this anyway just for show and to test it on the demo instance. Afterwards i will remove this code.